exchangelib/properties.py

```diff
@@ -117,33 +117,40 @@ def from_xml(cls, elem, account):
         elem.clear()
         return cls(**kwargs)
 
-    def to_xml(self, version):
+    def to_xml(self, version, exclude_read_only=True):
```

Comment:
So this is very much a semantic nitpick but what do you think about renaming the param `include_read_only_attributes` and set it to `False` by default?

To me, read-only attributes are something that you only need in some special cases, so they're not excluded by default — they're included when you need them.

Reply:
I flipped back/forth between `include_read_only_attributes` and `exclude_read_only_attributes`. i opted to used the verb `exclude` b/c I thought it'd read better when it was used...
```
if f.is_read_only and exclude_read_only is True:
                continue
```
vs
```
if f.is_read_only and include_read_only is False:
                continue
```

Maybe i'm in the minority here though lol but i found the second one kinda confusing to read when I wrote it out. Lmk if you have thoughts on that though!

I like appending `attributes` to the param though. I'll add that! thx

Reply:
Ah, it's your diff so it your call! However, if you find the second form hard to read, we can always create a temporary variable, like this:
```python
exclude_read_only = not include_read_only_attributes
if f.is_read_only and exclude_read_only is True:
                continue
```